lib/migrate/extractTextPlugin/__testfixtures__/extractTextPlugin.input.js

```diff
@@ -1,16 +1,26 @@
-let ExtractTextPlugin = require('extract-text-webpack-plugin');
-let HTMLWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 module.export = {
     module: {
         rules: [
             {
                 test: /\.css$/,
-                use: ExtractTextPlugin.extract('style-loader', 'css-loader')
+                use: ExtractTextPlugin.extract({
+                    use: 'css-loader',
+                    fallback: ['style-loader', { loader: 'test-object-loader' }],
```

Comment:
I tried to fully demonstrate that string, object, and array loader configurations all work via this test case.